pkg/minikube/cruntime/cruntime.go

```diff
@@ -47,6 +47,11 @@ func (cs ContainerState) String() string {
 	return [...]string{"all", "running", "paused"}[cs]
 }
 
+// ValidRuntimes lists the supported container runtimes
+func ValidRuntimes() []string {
+	return []string{"docker", "crio", "containerd"}
```

Comment:
The default spelling is supposed to be "cri-o", like in https://cri-o.io/

But we accept **both** as parameters (and normalize "crio" to "cri-o")

Reply:
Gotcha. Would you prefer that to be list in the docs as well? Currently, it has it spelt as `crio`:
`--container-runtime='docker': The container runtime to be used (docker, crio, containerd).`

Should that stay the same continue to silently accept `cri-o` or should `cri-o` be included in the docs?

Reply:
I think the doc should be one way but we should accept alternative spellings . Thank you for this contribution

Reply:
I'm still slightly confused. Just to clarify, the docs should stay the same (spelling `crio` as `cri-o`) but I should ensure that both spellings are accepted. Do I have that correct?

Reply:
sorry for the lack of clarity, 
the docs should say cri-o
and our help text should also say cri-o
but we should also accept it if the user provides crio as an option.

Reply:
Ah gotcha. Will update that!

Reply:
Upstream is confused as well (the project is called cri-o, the binary is called crio)

Reply:
I'm actually not sure if "Cry... Oh!" (CRI-O) is any better than "Ociieed!" (OCID)

Reply:
https://www.redhat.com/en/blog/introducing-cri-o-10